src/py/flwr/client/numpy_client.py

```diff
@@ -175,6 +189,10 @@ class NumPyClientWrapper(Client):
 
     def __init__(self, numpy_client: NumPyClient) -> None:
         self.numpy_client = numpy_client
+        self.properties: Properties = {"tensor_type": "numpy.ndarray"}
+
+    def get_properties(self, ins: PropertiesIns) -> PropertiesRes:
+        return PropertiesRes(properties=self.properties)
```

Comment:
Using self.properties here means that `NumPyClient.get_properties` never gets called. My recommendation is to keep `NumPyClientWrapper` stateless (apart from the `numpy_client` property). The most flexible approach would be to leave setting ` {"tensor_type": "numpy.ndarray"}` up to the user.

Reply:
I removed `self.properties` from wrapper and forwarded the `get_properties` call to `numpy_client` attribute.